prep-html-skeleton/index.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+  <head></head>
+  <title>HTML Skeleton Exercise</title>
```

Comment:
Hi Alvaro! Be sure to put your `<title></title>` inside of your `<head></head>` element

Reply:
<img width="1436" alt="Screen Shot 2022-04-18 at 4 12 25 PM" src="https://user-images.githubusercontent.com/103209321/163891766-854721fd-7da3-4b6f-be44-0db84b262afa.png">

https://github.com/alvarodiazjr/prep-0422-code-solutions/pull/1/commits/819b36a6f85e46b10ab0643220eafca2f62776df